OtripleS.Web.Api.Tests.Acceptance/APIs/TeacherContacts/TeacherContactsApiTests.cs

```diff
@@ -114,7 +114,7 @@ private static Filler<Teacher> CreateTeacherFiller()
 
             var filler = new Filler<Teacher>();
 
-            filler.Setup()
+            filler.Setup().OnProperty(teacher => teacher.Status).Use(TeacherStatus.Active)
```

Comment:
To keep it consistent, please move this line to next line.

Reply:
Moved as advised & noted for future commits!

> To keep it consistent, please move this line to next line.